src/cpp/CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ target_include_directories(${TARGET_NAME}
 
 target_include_directories(${TARGET_NAME} SYSTEM PRIVATE "${safetensors.h_SOURCE_DIR}")
 
-target_link_libraries(${TARGET_NAME} PUBLIC openvino::runtime PRIVATE openvino::threading nlohmann_json::nlohmann_json jinja2cpp)
+target_link_libraries(${TARGET_NAME} PUBLIC openvino::runtime PRIVATE openvino::threading nlohmann_json::nlohmann_json jinja2cpp ${CMAKE_DL_LIBS})
```

Comment:
`CMAKE_DL_LIBS` contains the name of the library that has `dlopen` and `dlclose` in it but there are no such calls in genai. What is it for?

Reply:
https://man7.org/linux/man-pages/man3/dladdr.3.html

Reply:
That didn't help

Reply:
![{4AE795DD-2BDC-46E2-8E92-8381D0938431}](https://github.com/user-attachments/assets/c2966af3-93b0-4dad-baf7-f661f0a8c1bf)